### PR TITLE
Home Work 5

### PR DESCRIPTION
**Pull request by Maksimka1289**

### CODE REVIEW

File: ALevel HW5/Action.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ALevel_HW5
+{
+    public class Actions
+    {
+        private Logger logger = Logger.Instance;
+
+        public Result Method1()
```
Comment (petrochernykh) [inline ALevel HW5/Action.cs:13]: You should use more descriptive naming. Like Success, Skipped, Failed smth with these words

File: ALevel HW5/Starter.cs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+                        if (!result.Status)
+                        {
+                            logger.Log("Error", "Action failed by a reason: " + result.ErrorMessage);
+                        }
+                        break;
+                }
+            }
+
+            string logContent = logger.GetLogs();
+            File.WriteAllText("log.txt", logContent);
```
Comment (petrochernykh) [inline ALevel HW5/Starter.cs:49]: Have you seen log.txt file?

Comment (Maksimka1289) [reply ALevel HW5/Starter.cs:49]: Yes, this file be on this path: ALevel HW5\bin\Debug\net7.0\log.txt

**Comment by Maksimka1289** [comment]
read

**Pull request closed without merging**